src/javascripts/helpers/auth.js

```diff
@@ -3,15 +3,18 @@ import 'firebase/auth';
 import loginButton from '../components/loginButton';
 import logoutButton from '../components/logoutButton';
 import firebaseConfig from './apiKeys';
+import startApp from './views/startApp';
 
 const checkLoginStatus = () => {
   firebase.initializeApp(firebaseConfig);
   firebase.auth().onAuthStateChanged((user) => {
     if (user) {
       // person is logged in do something...
+      startApp();
```

Comment:
You're calling this is various places. Make sure that you either call it here on in the main.js